learningresources/tests/test_api.py

```diff
@@ -61,6 +64,10 @@ def test_create(self):
             create_course(**self.kwargs)
         self.assertTrue(course_count() == after)
 
+        # NOT NULL constraint fails on org
+        with self.assertRaises(IntegrityError):
```

Comment:
Do they both contain the string "org"? It's better to match on a `contains` than nothing at all.

When is sqlite3 used? I thought it was always PostgreSQL, in all environments?